compiler/src/iree/compiler/Codegen/LLVMGPU/ConvertToROCDL.cpp

```diff
@@ -36,14 +40,32 @@ namespace iree_compiler {
 
 namespace {
 
+// Transform gpu.barrier -> amdgpu.lds_barrier
+struct RewriteBarriers  : public OpRewritePattern<gpu::BarrierOp> {
+  using OpRewritePattern<gpu::BarrierOp>::OpRewritePattern;
+
+  LogicalResult matchAndRewrite(gpu::BarrierOp op,
+                                PatternRewriter &rewriter) const override {
+    OpBuilder::InsertionGuard guard(rewriter);
+    rewriter.setInsertionPoint(op);
+    rewriter.replaceOpWithNewOp<amdgpu::LDSBarrierOp>(op);
+    return success();
+  }
+};
+
+void populatePrepareVectorToAMDMMAPatterns(RewritePatternSet &patterns) {
```

Comment:
The name of this pattern population function looks out of date. Maybe `populateConvertGPUToAMDGPUPatterns`?

Reply:
Agree with that.